programmers-coding-test/wonji/implement/왕실의나이트.js

```diff
@@ -0,0 +1,30 @@
+const rl = require("readline-sync");
+const input_data = rl.question("현재 나이트의 위치를 입력하시오.\n>>").toLowerCase();
+
+// console.log(input_data);
+
+const column = parseInt(input_data[0].charCodeAt(0)) - parseInt('a'.charCodeAt(0)) + 1;
+const row = parseInt(input_data[1]);
+
+// console.log(column);
+// console.log(row);
+
+const steps = [ [-2, -1], [-1, -2], [1, -2], [2, -1],
+[2, 1], [1, 2], [-1, 2], [-2, 1] ];
+
+// console.log(steps);
+// console.log(steps[0]);
+
+let result = 0;
+let next_row, next_column = 0;
```

Comment:
밑에서 next_row에 값을 대입해서 가능한 코드임.
이건 파이썬에서 가능한 문법임.
```suggestion
let [next_row, next_column] = [0, 0];
```